ceylon-mode.el

```diff
@@ -81,15 +81,16 @@
   "Syntax highlighting for Ceylon lowercase identifiers.")
 (defconst ceylon-font-lock-uidentifier
   (list
-   '("\\<\\([[:upper:]][[:alnum:]_]*\\)\\>" . font-lock-type-face)
-   '("\\<\\(\\\\I[[:alnum:]_]*\\)\\>" . font-lock-type-face))
+   '("\\<\\([[:upper:]][[:alnum:]_]*\\)\\>\\??" . font-lock-type-face)
+   '("\\<\\(\\\\I[[:alnum:]_]*\\)\\>\\??" . font-lock-type-face))
```

Comment:
I like this idea, but I think this isn’t the best way to implement it: this is the regex for UIdentifiers. I think it would be better to define a new constant for, dunno – perhaps  “type sigils”? – and add that to the big `append` in `ceylon-font-lock-all`.